apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -138,6 +159,20 @@ public final class AppConfiguration extends BaseAppConfiguration implements Seri
   public static final String ENV_VAR_KEY_RIF_JOB_QUEUE_SIZE_MULTIPLE =
       "RIF_JOB_QUEUE_SIZE_MULTIPLE";
 
+  /**
+   * The name of the environment variable that should be used to provide the work queue size for the
+   * RIF loader's thread pool when processing claims data. This number is multiplied by the number
+   * of worker threads to obtain the actual queue size. Lower sizes are more memory efficient but
+   * larger sizes could provide a performance improvement in some circumstances.
+   *
+   * <p>Benchmarking is necessary to determine an optimal value in any given environment. Generally
+   * smaller is better. The default value provides some slack for handling intermittent database
+   * slow downs without wasting too much RAM with large numbers of objects waiting to be sent to the
+   * database.
+   */
+  public static final String ENV_VAR_KEY_CLAIM_RIF_JOB_QUEUE_SIZE_MULTIPLE =
+      "RIF_JOB_QUEUE_SIZE_MULTIPLE_CLAIMS";
```

Comment:
I think this is probably an existing problem with the above naming scheme that was copied here, but this eventually gets used as a _multiplier_ but is named "multiple" as if we're only using this when we have >1 claims. Just a bit confusing, might be worth cleaning up if we're making a new version.

(See gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java:126 where the other value gets used eventually)

Reply:
This setting follows the same pattern as the existing setting used for benes.  I'd rather follow that convention to avoid a different type of confusion.

And rather than trigger a bunch of changes to terraform and settings files over a word choice issue maybe we create a different JIRA to address problematic setting names?

Reply:
Yeah that's fair, we can/should tackle them all at once and across the system. Created https://jira.cms.gov/browse/BFD-2756 for this.